definitions/contested/json/CaseEventToComplexTypes/CaseEventToComplexTypes-manageScannedDocs.json

```diff
@@ -4,64 +4,97 @@
     "CaseEventID": "FR_manageScannedDocs",
     "CaseFieldID": "manageScannedDocumentCollection",
     "ListElementCode": "caseDocumentType",
-    "DisplayContext": "MANDATORY"
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "manageScannedDocumentCollection.selectForUpdate=\"Yes\"",
+    "RetainHiddenField": "Yes"
   },
   {
     "LiveFrom": "01/06/2022",
     "ID": "FR_manageScannedDocs",
     "CaseEventID": "FR_manageScannedDocs",
     "CaseFieldID": "manageScannedDocumentCollection",
     "ListElementCode": "caseDocumentOther",
-    "DisplayContext": "MANDATORY"
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "manageScannedDocumentCollection.selectForUpdate=\"Yes\"",
+    "RetainHiddenField": "Yes"
   },
   {
     "LiveFrom": "01/06/2022",
     "ID": "FR_manageScannedDocs",
     "CaseEventID": "FR_manageScannedDocs",
     "CaseFieldID": "manageScannedDocumentCollection",
     "ListElementCode": "hearingDetails",
-    "DisplayContext": "OPTIONAL"
+    "DisplayContext": "OPTIONAL",
+    "FieldShowCondition": "manageScannedDocumentCollection.selectForUpdate=\"Yes\"",
+    "RetainHiddenField": "Yes"
   },
   {
     "LiveFrom": "01/06/2022",
     "ID": "FR_manageScannedDocs",
     "CaseEventID": "FR_manageScannedDocs",
     "CaseFieldID": "manageScannedDocumentCollection",
     "ListElementCode": "caseDocumentConfidentiality",
-    "DisplayContext": "MANDATORY"
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "manageScannedDocumentCollection.selectForUpdate=\"Yes\"",
+    "RetainHiddenField": "Yes"
   },
   {
     "LiveFrom": "01/06/2022",
     "ID": "FR_manageScannedDocs",
     "CaseEventID": "FR_manageScannedDocs",
     "CaseFieldID": "manageScannedDocumentCollection",
     "ListElementCode": "caseDocumentFdr",
-    "DisplayContext": "MANDATORY"
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "manageScannedDocumentCollection.selectForUpdate=\"Yes\" AND manageScannedDocumentCollection.caseDocumentConfidentiality=\"No\"",
+    "RetainHiddenField": "Yes"
   },
   {
     "LiveFrom": "01/06/2022",
     "ID": "FR_manageScannedDocs",
     "CaseEventID": "FR_manageScannedDocs",
     "CaseFieldID": "manageScannedDocumentCollection",
     "ListElementCode": "caseDocumentParty",
-    "DisplayContext": "MANDATORY"
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "manageScannedDocumentCollection.selectForUpdate=\"Yes\" AND manageScannedDocumentCollection.caseDocumentConfidentiality=\"No\" AND manageScannedDocumentCollection.caseDocumentFdr=\"No\" AND manageScannedDocumentCollection.caseDocumentType!=\"Without Prejudice offers\" AND manageScannedDocumentCollection.caseDocumentType!=\"Attendance Sheets\" AND manageScannedDocumentCollection.caseDocumentType!=\"Judicial notes\" AND manageScannedDocumentCollection.caseDocumentType!=\"Judgment\" AND manageScannedDocumentCollection.caseDocumentType!=\"Witness Summons\" AND manageScannedDocumentCollection.caseDocumentType!=\"Transcript\"",
```

Comment:
I honestly think this condition is too big and probably deserves a field for it set on aboutToStart callback